### PR TITLE
Fix/layout

### PR DESCRIPTION
**Pull request by Taloreff-hls**
Final layout fixes

Inifinite scroll bug fix

### CODE REVIEW

File: src/assets/globalStyles/GlobalStyle.tsx [hunk 1]
```diff
@@ -8,7 +8,7 @@ const GlobalStyle = createGlobalStyle`
     box-sizing: border-box;
     margin: 0;
     padding: 0;
-    height: 100vh;
+    min-height: 100vh;
```
Comment (DinFridman) [inline src/assets/globalStyles/GlobalStyle.tsx:12]: why did you change it to min?

File: src/cmps/FightArena.tsx [hunk 2]
```diff
@@ -148,11 +155,32 @@ const FightArena = ({ selectedPokemon, opponentPokemon }: FightArenaProps) => {
           isFightClicked={isFightClicked}
         />
         {!isFightClicked ? (
-          renderButton("Fight", handleFightClick)
+          renderButton(
+            "Fight",
+            handleFightClick,
+            colors.primary[300],
+            700,
```
Comment (DinFridman) [inline src/cmps/FightArena.tsx]: what is the 700?

File: src/cmps/FightArena.tsx [hunk 3]
```diff
@@ -148,11 +155,32 @@ const FightArena = ({ selectedPokemon, opponentPokemon }: FightArenaProps) => {
...
+            TypographyType.HeadingXxxxl,
+            "232px"
```
Comment (DinFridman) [inline src/cmps/FightArena.tsx]: maybe pass the props as sx prop or styles prop?

File: src/pages/PokemonContent.tsx [hunk 4]
```diff
@@ -23,22 +23,29 @@ const PokemonContent = ({ selectedCtg }: PokemonContentProps) => {
...
 
   const filteredPokemonData = useMemo(() => {
-    return pokemonData
+    const filteredData = pokemonData
       .filter((pokemon: Pokemon) =>
         pokemon.name.english.toLowerCase().includes(searchValue.toLowerCase())
       )
       .filter((pokemon: Pokemon) =>
         selectedCtg === 1 ? pokemon.belongsToUser : true
       );
```
Comment (DinFridman) [inline src/pages/PokemonContent.tsx]: can you reduce it to 1 filter with two conditions?

**Review by DinFridman — CHANGES_REQUESTED** [review]

File: src/assets/globalStyles/GlobalStyle.tsx (hunk 1 above)
Comment (Taloreff-hls) [reply src/assets/globalStyles/GlobalStyle.tsx:12]: Because of the infinite scroll i need more than 100vh

File: src/cmps/FightArena.tsx (hunk 2 above)
Comment (Taloreff-hls) [reply src/cmps/FightArena.tsx]: font weight

**Review by DinFridman — APPROVED** [review]
